From f36cb4e3bf250a8a91dfdec6056efa378551000c Mon Sep 17 00:00:00 2001
From: "Huang, Tao" <huangtao@rock-chips.com>
Date: Fri, 7 Oct 2016 11:59:39 +0800
Subject: [PATCH] extcon: remove EXTCON_PROP_USB_ID property

Fix commit 536277d5500a ("FROMLIST: extcon: Add the support for
extcon property according to extcon type"), which introduce
EXTCON_PROP_USB_ID property, but upstream don't have such property.
Remove it make merge easy.

Change-Id: I7905049629aa85158b7e705b40018f83fa85a9ac
Signed-off-by: Huang, Tao <huangtao@rock-chips.com>
---
 include/linux/extcon.h | 11 +++--------
 1 file changed, 3 insertions(+), 8 deletions(-)

diff --git a/include/linux/extcon.h b/include/linux/extcon.h
index 950f8491fa6a..ad7a1606a7f3 100644
--- a/include/linux/extcon.h
+++ b/include/linux/extcon.h
@@ -99,10 +99,6 @@
 /*
  * Properties of EXTCON_TYPE_USB.
  *
- * - EXTCON_PROP_USB_ID
- * @type:	integer (intval)
- * @value:	0 (low) or 1 (high)
- * @default:	0 (low)
  * - EXTCON_PROP_USB_VBUS
  * @type:	integer (intval)
  * @value:	0 (low) or 1 (high)
@@ -112,12 +108,11 @@
  * @value:	0 (normal) or 1 (flip)
  * @default:	0 (normal)
  */
-#define EXTCON_PROP_USB_ID		0
-#define EXTCON_PROP_USB_VBUS		1
-#define EXTCON_PROP_USB_TYPEC_POLARITY	2
+#define EXTCON_PROP_USB_VBUS		0
+#define EXTCON_PROP_USB_TYPEC_POLARITY	1
 
 #define EXTCON_PROP_USB_MIN		0
-#define EXTCON_PROP_USB_MAX		2
+#define EXTCON_PROP_USB_MAX		1
 #define EXTCON_PROP_USB_CNT	(EXTCON_PROP_USB_MAX - EXTCON_PROP_USB_MIN + 1)
 
 /* Properties of EXTCON_TYPE_CHG. */
-- 
2.35.3

